src/lwc/geTemplateBuilderFormFields/geTemplateBuilderFormFields.js

```diff
@@ -505,13 +541,12 @@ export default class geTemplateBuilderFormFields extends LightningElement {
     * component chain: geTemplateBuilderFormField -> geTemplateBuilderFormSection -> here
     */
     handleDeleteFormElement(event) {
-        const fieldMapping = TemplateBuilderService.fieldMappingByDevName[event.detail.fieldName];
         const element = this.template.querySelector(`lightning-input[data-field-mapping="${event.detail.fieldName}"]`);
+
         element.checked = false;
```

Comment:
@ptewson-sfdo - saw your message in slack and looked into it. I think we should wrap this line in a simple `if (element)` so that we don't attempt to uncheck the checkbox for form fields with deleted mappings. They won't be checked onload in any case. Long term I think we should really be disabling these checkboxes in the sidebar for broken field mappings.